tasks.py

```diff
@@ -341,6 +341,20 @@ def post_upgrade(context):
     run_command(context, command)
 
 
+@task()
```

Comment:
Can you please add `help` for `--format` in keeping with other task functions? See how it's done with `makemigrations` above. ☝️